src/js/navigation.js

```diff
@@ -2,12 +2,23 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", (e) => {
+  e.stopPropagation();
```

Comment:
This prevents the event from firing again on `.welcom`, because it's underneath the button.

Reply:
ok!